spec/rails/doc/openapi.json

```diff
@@ -105,46 +105,6 @@
           }
         },
         "parameters": [
-          {
-            "name": "page",
-            "in": "query",
-            "schema": {
-              "type": "integer"
-            },
-            "example": 1
-          },
-          {
-            "name": "per",
-            "in": "query",
-            "schema": {
-              "type": "integer"
-            },
-            "example": 10
-          },
-          {
-            "name": "X-Authorization-Token",
-            "in": "header",
-            "required": true,
-            "schema": {
-              "type": "string"
-            },
-            "example": "token"
-          },
-          {
-            "name": "filter[name]",
-            "in": "query",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "name": {
-                  "type": "string"
-                }
-              }
-            },
-            "example": {
-              "name": "Example Table"
-            }
-          },
```

Comment:
Could you explain the reason why `page`, `per`, `X-Authorization-Token`, and `filter[name]` need to be deleted?

Reply:
Those were deleted because the below logic was deleted.

```
if key == 'parameters'
          base[key] |= value
          base[key].uniq! { |param| param.slice('name', 'in') }
```

Reverted the change.

Reply:
Added a test case to ensure the base `parameters` are replaced with new spec. https://github.com/k0kubun/rspec-openapi/pull/55/commits/7bf6054ff36821201b842988562cb33cc015bd0e